unified_planning/model/state.py

```diff
@@ -36,7 +37,14 @@ class COWState(ROState):
     """This is an abstract class representing a classical `Read/Copy-on-Write state`"""
 
     def make_child(
-        self, updated_values: Dict["up.model.FNode", "up.model.FNode"]
+        self,
+        updated_values: Dict["up.model.FNode", "up.model.FNode"],
+        running_events: Optional[
+            List[List["up.engines.mixins.simulator.Event"]]
+        ] = None,
+        stn: Optional["up.model.delta_stn.DeltaSimpleTemporalNetwork"] = None,
+        durative_conditions: Optional[List["up.model.fnode.FNode"]] = None,
+        events: Optional[Set["up.engines.mixins.simulator.Event"]] = None,
```

Comment:
This new parameters are not documented and in general I don't think we want to introduce temporal state in this way.

Probably we want a `ROTemporalState`, a `COWTemporalState` and a `UPCOWTemporalState`.